tests/put/test_get_option_parser.py

```diff
@@ -8,56 +8,56 @@ def setUp(self):
         self.parser = get_option_parser("program-name", 'stdout', 'stderr')
 
     def test(self):
-        (options, args) = self.parser.parse_args([])
+        options = self.parser.parse_args(['file'])
```

Comment:
As a general rule we shouldn't modify the tests input in a refactor commit. If we do this we will be not sure if the behaviour is the same.
Please change this
```
options = self.parser.parse_args(['file'])
```
to this:
```
options = self.parser.parse_args([])
```

Please do the same for all other tests.

Reply:
If remove `'file'`, it will always display
```
Usage: trash-put [OPTION]... FILE...

trash-put: error: Please specify the files to trash.
```

Reply:
I thinks it's because you need first to set nargs to "*" in put.py

Reply:
Fixed.